extendedmodlog/eventmixin.py

```diff
@@ -1545,7 +1544,8 @@ async def on_member_update(self, before: discord.Member, after: discord.Member)
             member=before,
             m_id=before.id,
         )
-        emb_msg = _("{member} ({m_id}) updated").format(member=before, m_id=before.id)
+        embed.set_footer(text=_("User ID: ") + str(before.id))
+        emb_msg = _("{member} updated").format(member=before)
```

Comment:
This is alsoinconsistent with all the other logging such as member join logging it has the member ID after the members diuscord name.